src/test/ReactPerf.js

```diff
@@ -50,6 +65,22 @@ var ReactPerf = {
   },
 
   /**
+   * Version of measureMethods that doesn't mutate the object until perf
+   * measurement starts, which makes stack traces cleaner if we never start
+   * perf. Be careful to make sure that you don't accidentally call a copy of
+   * the un-instrumented function.
```

Comment:
Hmm, this bit concerns me. We copy the properties from `ReactCompositeComponent.Mixin` once - won't those be the uninstrumented functions?